massdriver-application/aws_identity.tf

```diff
@@ -33,7 +33,7 @@ locals {
 }
 EOF
 
-  aws_federated_principal_assume_role = !local.is_aws ? null : <<EOF
+  aws_federated_principal_assume_role = !local.is_aws && !local.is_kubernetes ? null : <<EOF
```

Comment:
I'm sure you're aware but dupe of another open PR: https://github.com/massdriver-cloud/terraform-modules/pull/36/files

Reply:
Closed it. Let’s roll lambda templates!!!